ui/src/common/containers/ReferenceListContainer.jsx

```diff
@@ -30,9 +30,14 @@ const stateToProps = (state) => ({
 });
 
 const dispatchToProps = (dispatch, ownProps) => ({
-  onQueryChange(query) {
+  onPageChange(page, size) {
```

Comment:
why we need to change props for the component?

Reply:
Similar reason to the above - after updating one new issue became visible, which is the page size was not changing when selected new page size. Turned out in antd pagination component the ```onChange``` fires whenever anything changes in the pagination, not only when changing page number. Meaning, when you change the page size, it triggers both```onShowSizeChange``` and ```onChange```. But ```onChange``` has higher priority hence it triggers last. So it sets the page back to the same one. That's why I added ```size``` to props as well, so that the page size change works properly again.